crypto/message.go

```diff
@@ -422,6 +374,66 @@ func (msg *PGPMessage) SeparateKeyAndData(estimatedLength, garbageCollector int)
 	return outSplit, nil
 }
 
+func readPacketContents(contents io.Reader, estimatedLength int, garbageCollector int) ([]byte, error) {
+	gcCounter := 0
+	// TODO: add support for multiple keypackets
```

Comment:
Nit: it was already not really in the right location, but this comment should probably stay in the previous function (e.g. under the first case).

Reply:
Done